### PR TITLE
PBS-64: Fix HTML-encoding errors

### PR DESCRIPTION
**Pull request by guscarreon**
Using the NewEncoder function that will help correct all '&', '<', and '>' characters.

### CODE REVIEW

**Comment by dbemiller** [comment]
One best practice I'd recommend: whenever you fix a bug, add a test which makes sure it doesn't come back again, and label it with a well-known comment string.

For example, if you search for `// Prevents #` in the project, you'll find some of the other regression tests for bugs we've had. If everyone's consistent about it, it's incredibly helpful. Cause the next time someone refactors the code, the test will break and they'll have a Github issue they can check with all the details.

**Comment by guscarreon** [comment]
David, I don't know the ```// Prevents #``` notation. In your review, if you have time to do one, could you please point out what format or comment conventions to follow on the test case function?

**Comment by dbemiller** [comment]
@guscarreon like this: https://github.com/prebid/prebid-server/blob/master/endpoints/openrtb2/amp_auction_test.go#L86

It's not an official notation or anything... just a convention I followed in case we wanted to find them all with `grep` or something like that.

File: exchange/exchange_test.go [hunk 1]
```diff
@@ -155,7 +155,8 @@ func TestCharacterEscape(t *testing.T) {
 		t.Errorf("exchange.buildBidResponse returned %d errors", len(errList))
 	}
 
-	if bytes.Contains(bid_resp.Ext, []byte("&")) {
+	//if bytes.Contains(bid_resp.Ext, []byte("&")) && !bytes.Contains(bid_resp.Ext, []byte("u0026")) {
```
Comment (hhhjort) [inline exchange/exchange_test.go]: Good fix, not sure what the purpose of this comment is?

**Pull request merged**